content/en/tutorials/integration-tutorials/lightning.md

```diff
@@ -318,4 +318,4 @@ I hope you find this report helpful. I will encourage to play with the code and
 
 Here are some resources to learn more about PyTorch Lightning:
 - [Step-by-step walk-through](https://lightning.ai/docs/pytorch/latest/starter/introduction.html) - This is one of the official tutorials. Their documentation is really well written and I highly encourage it as a good learning resource.
-- [Use Pytorch Lightning with Weights & Biases](https://wandb.me/lightning) - This is a quick colab that you can run through to learn more about how to use W&B with PyTorch Lightning.
\ No newline at end of file
+- [Use Pytorch Lightning with W&B](https://wandb.me/lightning) - This is a quick colab that you can run through to learn more about how to use W&B with PyTorch Lightning.
```

Comment:
Within a list item, use colons instead of dashes to separate a summary from more info.